x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/pki/PkiRealm.java

```diff
@@ -317,6 +317,17 @@ public void expireAll() {
         }
     }
 
+    @Override
+    public void usageStats(ActionListener<Map<String, Object>> listener) {
+        super.usageStats(ActionListener.wrap(stats -> {
+            stats.put("has_truststore", trustManager != null);
+            stats.put("has_delegated_realms", delegatedRealms != null);
```

Comment:
Can we avoid `delegated` here? I know the code calls it that, but the config is `authorization_realms`, and I worry we're over using the term `delegated` for this and proxy-pki

Reply:
Good suggestions, thank you :)